libs/opengl/src/CCylinder.cpp

```diff
@@ -59,7 +59,42 @@ void CCylinder::render_dl() const
 
 #endif
 }
-
+void CCylinder::serializeTo(mrpt::serialization::CSchemeArchiveBase& out) const
+{
+	out["datatype"] = std::string(this->GetRuntimeClass()->className);
```

Comment:
What about defining a MACRO (define) to avoid repeating this long line in every class?